pkg/dao/types/status/status_workflow_execution.go

```diff
@@ -26,6 +26,23 @@ var workflowExecutionStatusPaths = NewWalker(
 			WorkflowExecutionStatusCanceled,
 		},
 	},
+	func(d Decision[ConditionType]) {
+		d.Make(WorkflowExecutionStatusCanceled,
+			func(st ConditionStatus, reason string) *Summary {
+				switch st {
+				case ConditionStatusUnknown:
+					return &Summary{
+						SummaryStatus: "Canceling",
```

Comment:
Should we mark transitioning for canceling?

Reply:
updated